File: Sylvester Sequence/SylvesterSequence.py
#########################
## Sylvester Sequence  ##
#########################


#Example : 2 3 7 43 1807 3263443 ....

def seq(n):
        pdt = 1
        ans = 2
        N = 1000000007
        for i in range(1,n+1):
                ans = ((pdt % N) * (ans % N)) % N
                pdt = ans
                ans = (ans + 1) % N
        return ans

find_val = int(input("Enter the nth value to find : "))
print(find_val,"th value in Sylvester's Sequence is ",seq(find_val))

File: Sylvester Sequence/SylvesterSequence.java
import java.util.Scanner;

public class SylvesterSequence {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.print("Enter the nth value to find : ");
        int n = scan.nextInt();
        System.out.println(n + "th value in Sylvester Sequence is :" + seq(n));
    }

    private static int seq(int n) {
        int pdt = 1, ans = 2, N = 1000000007;
        for(int i=1; i<n+1; i++){
            ans = ((pdt % N) * (ans % N)) % N;
            pdt = ans;
            ans = (ans + 1) % N;
        }
        return ans;
    }
}
